wpilib/wpilib/analoginput.py

```diff
@@ -326,10 +326,16 @@ def pidGet(self):
     def getSmartDashboardType(self):
         return "Analog Input"
 
+    def initTable(self, subtable):
+        if subtable is not None:
+            self._valueEntry = subtable.getValue("Value")
```

Comment:
getValue != getEntry... I think getValue is equivalent to ntproperty.

Reply:
Ugh how many times did I do that...